README.md

```diff
@@ -23,31 +23,21 @@ As for the name: Sinatra + Warp = Scotty.
 
 ## Examples
 
-Run /basic.hs to see Scotty in action:
+Run `examples/basic.hs` to see Scotty in action (if you haven't, run `cabal update` first):
 
-```bash
-runghc examples/basic.hs
 ```
-`Setting phasers to stun... (port 3000) (ctrl-c to quit)`
-
-Or equivalently with [`stack`](https://docs.haskellstack.org/en/stable/):
-
-```bash
-stack exec -- scotty-basic
+$ cabal run scotty-basic
+Setting phasers to stun... (port 3000) (ctrl-c to quit)
```

Comment:
This command does not work when I test it in a recent fresh `haskell` Docker container. `stack run [...]` does, but only if a `stack.yaml` is present. If we keep this I'd suggest adding a `stack.yaml` file too so that it works out of the box, thoughts (see #390)?

Reply:
No, adding a `stack.yaml` is not a good practice because it forces a fixed resolver on the users.

Indeed, I agree that the current stack example is not self-contained because it assumes a `stack init && stack build`.

Reply:
`stack run -- scotty-basic` works without `stack build` before but still requires a `stack.yaml`. I can try whether `stack init && stack run ...` works though. I guess for their own projects people can use whichever tool they want anyways.